ui/src/chat/ChatScroller/ChatScroller.tsx

```diff
@@ -239,7 +276,7 @@ export default function ChatScroller({
           overflowY: 'scroll',
         }}
         atBottomThreshold={250}
-        atTopThreshold={2500}
```

Comment:
I had set this really high to try and have the top get loaded really early so your scrolling is never interrupted by loading, but maybe you found an issue?

Reply:
Ah yeah, this was leftover from trying to get the darn thing to work. It has been restored in the latest commit

Reply:
@arthyn FYI added some mobile specific values to account for smaller page size; also tweaked the overscan. LMK what you think!